group_vars/all/vars.yml

```diff
@@ -1,3 +1,23 @@
 ---
 # The template directory (as a relative path from the top of repository)
 templates_dir: 'templates/'
+# standard variable for all repositories that use github
```

Comment:
This looks much better, glad you were able to move so many things into the common variables.